### PR TITLE
Custom files for new and set

### PR DESCRIPTION
**Pull request by schaudhari6254888**
## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/ServiceBus/ServiceBus.Autorest/custom/New-AzServiceBusNamespaceV2.ps1 [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+Gets the SASKey of a ServiceBus namespace, queue or topic.
+.Description
+Gets the SASKey of a ServiceBus namespace, queue or topic.
```
Comment (damodaravadhani) [inline src/ServiceBus/ServiceBus.Autorest/custom/New-AzServiceBusNamespaceV2.ps1:19]: Change description here. @schaudhari6254888

File: src/ServiceBus/ServiceBus.Autorest/custom/New-AzServiceBusNamespaceV2.ps1 [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+#>
+
+function New-AzServiceBusNamespaceV2{
+	[OutputType([Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Models.Api202201Preview.IAccessKeys])]
+    [CmdletBinding(DefaultParameterSetName = 'SetExpanded', PositionalBinding = $false, ConfirmImpact = 'Medium')]
+	param(
+        [Parameter(ParameterSetName = 'NewExpandedNamespace', Mandatory, HelpMessage = "The name of ServiceBusNamespace")]
```
Comment (damodaravadhani) [inline src/ServiceBus/ServiceBus.Autorest/custom/New-AzServiceBusNamespaceV2.ps1:26]: ChangeTo CreateExpanded

File: src/ServiceBus/ServiceBus.Autorest/custom/New-AzServiceBusNamespaceV2.ps1 [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+<#
+.Synopsis
+Gets the SASKey of a ServiceBus namespace, queue or topic.
+.Description
+Gets the SASKey of a ServiceBus namespace, queue or topic.
+#>
+
+function New-AzServiceBusNamespaceV2{
+	[OutputType([Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Models.Api202201Preview.IAccessKeys])]
+    [CmdletBinding(DefaultParameterSetName = 'SetExpanded', PositionalBinding = $false, ConfirmImpact = 'Medium')]
```
Comment (damodaravadhani) [inline src/ServiceBus/ServiceBus.Autorest/custom/New-AzServiceBusNamespaceV2.ps1:24]: Change to CreateExpanded which should be the DefaultParameter set

File: src/ServiceBus/ServiceBus.Autorest/custom/New-AzServiceBusNamespaceV2.ps1 [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+<#
+.Synopsis
+Gets the SASKey of a ServiceBus namespace, queue or topic.
+.Description
+Gets the SASKey of a ServiceBus namespace, queue or topic.
+#>
+
+function New-AzServiceBusNamespaceV2{
+	[OutputType([Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Models.Api202201Preview.IAccessKeys])]
```
Comment (damodaravadhani) [inline src/ServiceBus/ServiceBus.Autorest/custom/New-AzServiceBusNamespaceV2.ps1:23]: Return type is not AccessKeys. Please Find the correct type.

File: src/ServiceBus/ServiceBus.Autorest/custom/Set-AzServiceBusNamespaceV2.ps1 [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+Gets the SASKey of a ServiceBus namespace, queue or topic.
+.Description
+Gets the SASKey of a ServiceBus namespace, queue or topic.
+#>
+
+function Set-AzServiceBusNamespaceV2{
+	[OutputType([Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Models.Api202201Preview.IAccessKeys])]
+    [CmdletBinding(DefaultParameterSetName = 'SetExpanded', PositionalBinding = $false, ConfirmImpact = 'Medium')]
+	param(
+        [Parameter(ParameterSetName = 'NewExpandedNamespace', Mandatory, HelpMessage = "The name of ServiceBusNamespace")]
```
Comment (damodaravadhani) [inline src/ServiceBus/ServiceBus.Autorest/custom/Set-AzServiceBusNamespaceV2.ps1:26]: SetExpandedNamespace is what the Parameterset name must be

**Comment by schaudhari6254888** [comment]
Closing this PR as this branch is based with main branch and Created the new PR https://github.com/Azure/azure-powershell/pull/20107

**Pull request closed without merging**